app/services/programme_type_mappings.rb

```diff
@@ -29,5 +29,41 @@ def training_programme(training_programme:)
         training_programme
       end
     end
+
+    def training_programme_friendly_name(name, short: false)
+      long_names = if mappings_enabled?
```

Comment:
Would this logic make more sense in a view helper called directly from the view component its used in, instead of in the service/`InductionRecord` model? the copy could even go in the translations file so we're not redefining the mappings, something like:

```
en:
  training_programme:
    short:
      full_induction_programme: FIP
      provider_led: Provider led
      ...
    long:
      full_induction_programme: Full induction programme
      provider_led: Provider led
      ...
```

Then the helper:

```
def training_programme_friendly_name(training_programme, length: :long)
  translation_key = ProgrammeTypeMapping.training_programme(training_programme:)
  I18n.t("training_programme_#{length}_#{translation_key}")
end
```

Reply:
Tks @ethax-ross I used your suggestion but kept it in the service/model as the same method is used in a serializer as well.. not only in views.